[wip] Make Content Less Overwhelming (#388)
ritterim/ritterim.github.io: _posts/2019-09-06-ux-ui-solutions-make-content-less-overwhelming.md

---
layout: post
title: "UX/UI Solutions: Make Content Less Overwhelming."
date: 2019-09-06 10:17:07
tags:
- UI/UX
- frontend
- SEO
categories:
- design
twitter_text: UX/UI Solutions. Make Content Less Overwhelming.
authors: Natasha Kurus
image: /images/ux-ui-solutions/ux-ui-solutions-banner.jpg
---

*How do you not overwhelm the user with a wall of content?*

When thinking about user experience, this question often comes to mind as we develop websites. The main goal for the UX/UI team is to guide visitors through the business in a way that shows exactly what you want them to see and to understand, all while delivering a better experience.

> For example, recently Marketing sent us four-pages of content to incorporate into the company website’s Support page. The content was split into four areas. However, those subjects had the same purpose – to promote the Support department on the website. Why not combine them and make the whole page aesthetically pleasing?

Reviewer: ```suggestion
Recently, our marketing department sent us four pages of content to incorporate into the company website’s Support page. The content was split into four areas. However, those subjects had the same purpose – to promote the Support department on the website. I asked myself, "why not combine them and make the whole page aesthetically pleasing?"
```

Our team decided to create one page with four custom icons, like tabs. It turned out really well! The page is now more interesting, more interactive, and much more appealing to the user.

[Here is](https://www.ritterim.com/support/) what we’ve got:
![Result](/images/ux-ui-solutions/ritterim-support.gif "Ritter Insurance Marketing Support")

> It’s worth mentioning the SEO terms, since those fields are hidden now. No worries. Youmay already know that Google rolled out the [Mobile First indexing initiative](https://webmasters.googleblog.com/2018/03/rolling-out-mobile-first-indexing.html) in March of 2018. As far as SEO ranking, the hidden content is treated equally. It’s safe to use tabbed content, sliding menus, accordions, and other similar methods to make content less overwhelming.

Reviewer: ```suggestion
It’s worth mentioning the SEO terms since those fields are hidden now. No worries. You may already know that Google rolled out the [Mobile First indexing initiative](https://webmasters.googleblog.com/2018/03/rolling-out-mobile-first-indexing.html) in March of 2018. As far as SEO ranking, the hidden content is treated equally. It’s safe to use tabbed content, sliding menus, accordions, and other similar methods to make content less overwhelming.
```

Author: @khalidabuhakmeh thank you!
